crates/autopilot/src/decoded_settlement.rs

```diff
@@ -670,12 +674,72 @@ mod tests {
             .total_surplus(
                 &external_prices,
                 &settlement.trades.iter().map(|t| t.order_uid).collect(),
+                &HashSet::new(),
```

Comment:
why not to pass here `eligible_owners`? :thinking:

Reply:
Not sure what to pass. In this test settlement contains regular user order. 

At real callsite eligible owners are fetched from db.